nucypher/cli/commands/cloudworkers.py

```diff
@@ -67,33 +71,89 @@ def up(general_config, staker_options, config_file, cloudprovider, aws_profile,
     STAKEHOLDER = staker_options.create_character(emitter, config_file)
 
     stakers = STAKEHOLDER.get_stakers()
-    if not stakers:
+    if not stakers and not create_unstaked:
         emitter.echo("No staking accounts found.")
         return
 
     staker_addresses = filter_staker_addresses(stakers, stakes)
 
     config_file = config_file or StakeHolderConfiguration.default_filepath()
 
-    deployer = CloudDeployers.get_deployer(cloudprovider)(emitter, STAKEHOLDER, config_file, remote_provider, nucypher_image, seed_network, sentry_dsn, aws_profile, prometheus)
-    config = deployer.create_nodes_for_stakers(staker_addresses)
+    deployer = CloudDeployers.get_deployer(cloudprovider)(emitter, STAKEHOLDER, config_file, remote_provider, nucypher_image, seed_network, sentry_dsn, aws_profile, prometheus, namespace=namespace, network=STAKEHOLDER.network)
+    if staker_addresses:
+        config = deployer.create_nodes(staker_addresses)
 
     if config.get('instances') and len(config.get('instances')) >= len(staker_addresses):
         emitter.echo('Nodes exist for all requested stakes', color="yellow")
         deployer.deploy_nucypher_on_existing_nodes(staker_addresses, wipe_nucypher=wipe)
 
 
+@cloudworkers.command('create')
+@click.option('--cloudprovider', help="aws or digitalocean", default='aws')
+@click.option('--aws-profile', help="The cloud provider account profile you'd like to use (an aws profile)", default=None)
```

Comment:
Should we make `aws-profile` more general like `cloud-profile` or something similar?